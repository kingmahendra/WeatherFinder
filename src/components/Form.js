import React from 'react'

export const Form = props => (
    <div className="form"> 
        <form onSubmit={props.getWeather}> 
        <input type="text" name="city" placeholder="city..."/>
        <input type="text" name="country" placeholder="country..."/>
        <button type="submit" className="btn btn-primary"> Find Weather </button>
        </form>
    </div>
)