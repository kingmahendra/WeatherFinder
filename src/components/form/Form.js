import React from 'react'

export const Form = props => (
    <div> 
        <form onSubmit={props.getWeather}> 
        <div>
            <input type="text" name="city" placeholder="City..."/>
            <input type="text" name="country" placeholder="Country..."/>
        </div>
        <button type="submit" className="btn btn-primary"> Find Weather </button>
        <button type="reset" onClick={props.reset} className="btn"> Reset</button>
        </form>
        
    </div>
)