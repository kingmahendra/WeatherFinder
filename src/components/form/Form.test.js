import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import { Form } from './Form'

describe('Form Component', ()=> {
    it('should render Form', () => {
        const component = shallow(<Form />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})