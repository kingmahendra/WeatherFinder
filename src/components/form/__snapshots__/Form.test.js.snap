// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render Form 1`] = `
<div>
  <form>
    <div>
      <input
        name="city"
        placeholder="City..."
        type="text"
      />
      <input
        name="country"
        placeholder="Country..."
        type="text"
      />
    </div>
    <button
      className="btn btn-primary"
      type="submit"
    >
       Find Weather 
    </button>
    <button
      className="btn"
      type="reset"
    >
       Reset
    </button>
  </form>
</div>
`;
